client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -119,6 +120,12 @@ const PluginsBrowser = ( {
 	hideHeader,
 	doSearch,
 } ) => {
+	const {
+		isAboveElement,
+		targetRef: searchHeaderRef,
+		referenceRef: navigationHeaderRef,
+	} = useScrollAboveElement();
```

Comment:
Should we consider replace our custom `use-scroll-above-element` to the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)?

It's pretty reliable right now: 

![image](https://user-images.githubusercontent.com/1044309/162076109-d027c4ae-2a4c-421a-b161-a0e37a9e6dd4.png)

Reply:
This is a good follow-up for 20% time 👍🏼  thanks @paulopmt1 !